indigo_api/models/documents.py

```diff
@@ -285,7 +285,7 @@ def expression_frbr_uri(self):
 
     @property
     def commencement_date(self):
-        return self.work.commencement_date
+        return self.work.main_commencement_date()
```

Comment:
We could do something similar on the work model, then it would retain the old API and we wouldn't have to change all the references to `work.commencement_date`.